prime-router/src/main/kotlin/tokens/SenderUtils.kt

```diff
@@ -13,30 +13,45 @@ import java.util.UUID
 class SenderUtils {
 
     companion object {
-        /**
-         * Generate a signed JWT, representing a request for authentication from a Sender, using a private key.
-         * This is done by the Sender, not by ReportStream. This method is here for testing, and as an example.
-         */
-        fun generateSenderToken(
-            sender: Sender,
+
+        fun generateToken(
```

Comment:
Github is showing that the kdoc disappeared, is that true?

Reply:
Discussed in a video chat, but this is to support exercising the test case where auth fails because the `jti` is missing